### PR TITLE
use latest 3dt

### PR DESCRIPTION
**Pull request by darkonie**
- update history service location
https://github.com/dcos/3dt/pull/45

- make 3dt find flapping systemd units
https://github.com/dcos/3dt/pull/44

### CODE REVIEW

**Comment by cmaloney** [comment]
@darkonie Is there any way we can test to make sure things like finding the history service / integration works?

**Comment by darkonie** [comment]
@cmaloney i have an idea of embedding a set of self tests which can be triggered by a CLI flag. What do you think about it? The issue is that under normal circumstances it is difficult to catch that kind of bugs

**Comment by malnick** [comment]
@cmaloney - @darkonie is going to implement a /info endpoint here so we can catch all the configuration 3DT has when running, we can use that to detect how it's getting data such as metrics which might be any of 3 sources.

**Comment by malnick** [comment]
Can we get a WIP label on this PR?

**Comment by darkonie** [comment]
@cmaloney @spahl ready to review

File: gen/calc.py [hunk 1]
```diff
@@ -380,7 +380,7 @@ def validate_os_type(os_type):
                 "prefix": 26                            \
               }                                         \
             ]}',
-        'rexray_config_method': 'empty'
+        'rexray_config_method': 'disabled'
```
Comment (cmaloney) [inline gen/calc.py]: this is old / current is still 'empty' to not break backwards compatibility.

**Comment by spahl** [comment]
:ship:

**Pull request merged**